tools/thinpool.sh

```diff
@@ -83,12 +83,12 @@ else
 
     pool_create() {
         echo sudo lvcreate --type thin-pool \
-             --poolmetadatasize 16GiB \
-             --size 1G \
+             --poolmetadatasize 16MiB \
```

Comment:
Just curious, why do we want to shrink the amount of pool size? from 16GiB to 16MiB seems a huge change.

Reply:
Not really sure the right size, but allocating 16GiB for metadata while allocating 1GB for data seems excessive.

Reply:
@kzys do you have an intuition what these numbers should be set to in a close-to-production setting? any rules of thumb?